Tools/cases_generator/generate_cases.py

```diff
@@ -986,6 +986,9 @@ def write_metadata(self) -> None:
             self.out.write_raw(self.from_source_files())
             self.out.write_raw(f"// Do not edit!\n")
 
+            self.out.write_raw("\n")
+            self.out.write_raw("#include <stdbool.h>")
```

Comment:
Why do you add <stdbool.h> include?

Reply:
Because there are some `bool` inside the `generate_metadata.h` file which, without this includes, doesn't compile. Previously, it was because we included stdbool.h before including this file so it was impliticitly available (see https://github.com/python/cpython/actions/runs/13614386994/job/38055588576?pr=130757)